MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+* To collapse all the groups.
+* To find the sorted or grouped [DataSource.DisplayItems](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataSource.DataSource.html#Syncfusion_Maui_DataSource_DataSource_DisplayItems) of underlying bound data to `SfListView`.
+
+### Tapped event
+
+The [ItemTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTapped) event will be triggered whenever tapping the item. Here, [TapCommandParameter](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_TapCommandParameter) property sets the parameter for [SfListView.TapCommand](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_TapCommand) and its default value is [ItemTappedEventArgs](). `ItemTappedEventArgs` has the following members which provides the information for `ItemTapped` event:
+
+ * [ItemType](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemTappedEventArgs.html#Syncfusion_Maui_ListView_ItemTappedEventArgs_ItemType): It gets the type of the tapped item.
+ * [ItemData](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemTappedEventArgs.html#Syncfusion_Maui_ListView_ItemTappedEventArgs_ItemData): The underlying data associated with the tapped item as its arguments.
+ * [Position](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemTappedEventArgs.html#Syncfusion_Maui_ListView_ItemTappedEventArgs_Position): Gets the touch position in the tapped item.
+
+{% tabs %}
+{% highlight c# %}
+
+listView.ItemTapped += ListView_ItemTapped;
+
+private void ListView_ItemTapped(object sender, Syncfusion.Maui.ListView.ItemTappedEventArgs e)
+{
+    if (e.ItemData == viewModel.InboxInfo[0])
+      viewModel.InboxInfo.Remove(e.ItemData as ListViewInboxInfo);  
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+The [ItemTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTapped) event is used for the following use cases:
+
+* To show the context menu.
+* To navigate to another page.
+* To delete the item in the list view at runtime.
+* To display the item details into another view.
+* To expand the view like, accordion.
+* To change the underlying bound data.
+* To skip other events like selection events if the Handled property set to `true`.
+
+### ItemDoubleTapped event
+
+The [ItemDoubleTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemDoubleTapped) event will be triggered whenever double tapping the item. The [ItemDoubleTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemDoubleTappedEventArgs.html) has the following members providing information for the `ItemDoubleTapped` event:

```

Comment:
The [ItemDoubleTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.ItemDoubleTappedEventArgs.html) has the following members which provide information for the `ItemDoubleTapped` event:

Reply:
I have modified the line as per the concerns provided